setup/native-script.txt

```diff
@@ -0,0 +1,24 @@
+
+# A boxter script to setup Windows machine for NativeScript development
```

Comment:
set up, also not sure about the boxter part? If this is a name, this should probably capitalized?